CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## 6.36.0 (Unreleased)
+## 6.36.1 (Unreleased)
+
+BUG FIXES: 
+* compute: fixed forced instance recreation when adding a `attached_disk` with `force_attach` being `false` to `google_compute_instance` ([#10064](https://github.com/hashicorp/terraform-provider-google-beta/pull/10064))
```

Comment:
I think the bug was when force_attach was unset, rather than when it was false?

Reply:
Updated the change log in the 6.36.1 branch.

https://github.com/hashicorp/terraform-provider-google-beta/commit/7d8781f33e4440ebcb427f9711e2d24a82dece86